test/e2e/utils.go

```diff
@@ -223,7 +223,7 @@ func run(node string, command ...string) (string, error) {
 	cmd.Stderr = &stderr
 	cmd.Stdout = &stdout
 	err := cmd.Run()
-	GinkgoWriter.Write([]byte(stdout.String() + stderr.String() + "\n"))
+	GinkgoWriter.Write([]byte(fmt.Sprintf("stdout: %.500s...\n, stderr %s\n", stdout.String(), stderr.String())))
```

Comment:
do you think it would be worth setting this via an environment variable?

Reply:
Let's do like this, in case of error vlans are show in full so we are good, another option is to dump to files the full output and print the truncated ones.